tests/testthat/test-rstudio_markers.R

```diff
@@ -100,11 +100,9 @@ test_that("it returns an empty list of markers if there are no lints", {
 
 test_that("rstudio_source_markers apply to print within rstudio", {
   withr::local_options(lintr.rstudio_source_markers = TRUE)
-  # TODO(@michaelchirico): Recent (as of this writing) withr v2.5.0 supports local_tempfile(lines = l) to simplify this
-  tmp <- withr::local_tempfile()
-  writeLines("1:ncol(x)", tmp)
-  empty <- withr::local_tempfile()
-  file.create(empty)
+
+  tmp <- withr::local_tempfile(lines = "1:ncol(x)")
+  empty <- withr::local_tempfile(lines = "")
```

Comment:
Wouldn't that be a file containing a single newline character instead of an empty file?

Reply:
Good catch.

Fixed it:

``` r
empty <- withr::local_tempfile()
file.create(empty)
#> [1] TRUE
readLines(empty)
#> character(0)

readLines(withr::local_tempfile(lines = character(0L)))
#> character(0)
```

<sup>Created on 2022-06-18 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>